scripts/python/services/test_GithubService.py

```diff
@@ -302,5 +303,55 @@ def test_throws_exception_when_client_throws_exception(self, mock_github_client_
             ConnectionError, github_service.create_new_team_with_repository, "test_team", "test_repository")
 
 
+@patch("gql.transport.aiohttp.AIOHTTPTransport.__new__", new=MagicMock)
+@patch("gql.Client.__new__", new=MagicMock)
+@patch("github.Github.__new__")
+class TestGithubServiceTeamExists(unittest.TestCase):
+    mock_unicorn_team = None
+    mock_super_team = None
+
+    def setUp(self):
+        self.mock_unicorn_team, self.mock_super_team = Mock(Team), Mock(Team)
+        # `name` is a reserved value in `Mock()` constructors. So need to mock the values manually.
+        self.mock_unicorn_team.name = "unicorn,team"
```

Comment:
Why add , and / in the team name string for?

Reply:
Just have a habit of adding random characters in strings, just in case something has a hard time handling them 😂 

Rather be more certain in the tests than find out in production, but usually not a scenario that grants it's own test case. 

But if people look with confusion, it may be more harmful than helpful! Don't want to be adding to [WTFs/minute](https://www.software.ac.uk/blog/2016-10-06-wtfsmin-indicator-code-quality) 😂